app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,7 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :configuration_script_authentications, :through => :configuration_scripts, :source => :authentications
```

Comment:
> This relation is there in ExtManagementSystem through the inclusion of AuthenticationMixin

sorry, what do you mean exactly? I dont see `configuration_script_authentications` there..